### ISSUE TITLE
Building from source on a git repository do not use selected tag on first run

### ISSUE THREAD

**Issue by sapetnioc** [prefix-dev/rattler-build#709]
rattler-build version: 0.13.0 installed from conda-forge

Running twice the building of the following recipe provides two different source directories. The first time, the source files correspond to the `master` (i.e. default) git branch whereas the second time source directory is on the requested tag. As long as the source cache exists in the output directory, the source directory is on the appropriate tag. If the cache is removed, the `master`branch is selected again.

Recipe file `/tmp/recipe/recipe.yaml`:
```
package:
    name: libminc
    version: 2.3.0

source:
  git: https://github.com/BIC-MNI/libminc
  tag: libminc-2-3-00

requirements:
  build:
    - git

build:
  number: 0
  script: |
    # Just print the current git status and the content of the source directory
    # to see differences in file size (e.g. on CMakeList.txt)
    git -C "$SRC_DIR" status
    ls -la "$SRC_DIR"
```
Running twice the following command on an unexisting output directory shows two different content for the source directory:
```
$ rattler-build build -r /tmp/recipe/ --output-dir /tmp/output
 Target platform: linux-64

 ╭─ Finding outputs from recipe
 │ Found 1 variants
 │ Build variant: libminc-2.3.0-hb0f4dca_0
 │ ╭─────────────────┬──────────╮
 │ │ Variant         ┆ Version  │
 │ ╞═════════════════╪══════════╡
 │ │ target_platform ┆ linux-64 │
 │ ╰─────────────────┴──────────╯
 │
 ╰─────────────────── (took 0 seconds)

 ╭─ Running build for recipe: libminc-2.3.0-hb0f4dca_0
 │
 │ ╭─ Fetching source code
 │ │ Fetching source from git repo: https://github.com/BIC-MNI/libminc
 │ │ Checked out revision: 'refs/tags/libminc-2-3-00' at '4ec4e79f7e7cf41ec61e24865096316efbf90bb3'
 │ │
 │ ╰─────────────────── (took 4 seconds)
 │
 │ ╭─ Resolving environments
 │ │ 
 │ │ Resolving environment for:
 │ │   Platform: linux-64
 │ │   Channels: 
 │ │    - /tmp/output
 │ │    - conda-forge
 │ │   Specs:
 │ │    - git
 │ │     /tmp/output/noarch   [00:00:00] Using cache
 │ │ 
 │ │ ╭─────────────────┬──────────────┬──────────────────┬─────────────┬────────────╮
 │ │ │ Package         ┆ Version      ┆ Build            ┆ Channel     ┆ Size       │
 │ │ ╞═════════════════╪══════════════╪══════════════════╪═════════════╪════════════╡
 │ │ │ git             ┆ 2.44.0       ┆ pl5321h709897a_0 ┆ conda-forge ┆ 10.21 MiB  │
 │ │ │ pcre2           ┆ 10.43        ┆ hcad00b1_0       ┆ conda-forge ┆ 928.56 KiB │
 │ │ │ openssl         ┆ 3.2.1        ┆ hd590300_0       ┆ conda-forge ┆ 2.73 MiB   │
 │ │ │ libexpat        ┆ 2.6.1        ┆ h59595ed_0       ┆ conda-forge ┆ 71.85 KiB  │
 │ │ │ libzlib         ┆ 1.2.13       ┆ hd590300_5       ┆ conda-forge ┆ 60.14 KiB  │
 │ │ │ libiconv        ┆ 1.17         ┆ hd590300_2       ┆ conda-forge ┆ 689.23 KiB │
 │ │ │ bzip2           ┆ 1.0.8        ┆ hd590300_5       ┆ conda-forge ┆ 248.27 KiB │
 │ │ │ gettext         ┆ 0.21.1       ┆ h27087fc_0       ┆ conda-forge ┆ 4.12 MiB   │
 │ │ │ libgcc-ng       ┆ 13.2.0       ┆ h807b86a_5       ┆ conda-forge ┆ 752.45 KiB │
 │ │ │ _libgcc_mutex   ┆ 0.1          ┆ conda_forge      ┆ conda-forge ┆ 2.50 KiB   │
 │ │ │ _openmp_mutex   ┆ 4.5          ┆ 2_gnu            ┆ conda-forge ┆ 23.07 KiB  │
 │ │ │ libgomp         ┆ 13.2.0       ┆ h807b86a_5       ┆ conda-forge ┆ 409.91 KiB │
 │ │ │ perl            ┆ 5.32.1       ┆ 7_hd590300_perl5 ┆ conda-forge ┆ 12.73 MiB  │
 │ │ │ libxcrypt       ┆ 4.4.36       ┆ hd590300_1       ┆ conda-forge ┆ 98.04 KiB  │
 │ │ │ curl            ┆ 8.5.0        ┆ hca28451_0       ┆ conda-forge ┆ 92.67 KiB  │
 │ │ │ zstd            ┆ 1.5.5        ┆ hfc55251_0       ┆ conda-forge ┆ 532.42 KiB │
 │ │ │ libssh2         ┆ 1.11.0       ┆ h0841786_0       ┆ conda-forge ┆ 264.78 KiB │
 │ │ │ libcurl         ┆ 8.5.0        ┆ hca28451_0       ┆ conda-forge ┆ 380.04 KiB │
 │ │ │ krb5            ┆ 1.21.2       ┆ h659d440_0       ┆ conda-forge ┆ 1.31 MiB   │
 │ │ │ keyutils        ┆ 1.6.1        ┆ h166bdaf_0       ┆ conda-forge ┆ 115.07 KiB │
 │ │ │ libnghttp2      ┆ 1.58.0       ┆ h47da74e_1       ┆ conda-forge ┆ 617.12 KiB │
 │ │ │ libev           ┆ 4.33         ┆ hd590300_2       ┆ conda-forge ┆ 110.12 KiB │
 │ │ │ libedit         ┆ 3.1.20191231 ┆ he28a2e2_2       ┆ conda-forge ┆ 120.97 KiB │
 │ │ │ c-ares          ┆ 1.27.0       ┆ hd590300_0       ┆ conda-forge ┆ 159.74 KiB │
 │ │ │ ncurses         ┆ 6.4          ┆ h59595ed_2       ┆ conda-forge ┆ 863.71 KiB │
 │ │ │ libstdcxx-ng    ┆ 13.2.0       ┆ h7e041cc_5       ┆ conda-forge ┆ 3.66 MiB   │
 │ │ │ ca-certificates ┆ 2024.2.2     ┆ hbcca054_0       ┆ conda-forge ┆ 151.79 KiB │
 │ │ ╰─────────────────┴──────────────┴──────────────────┴─────────────┴────────────╯
 │ │  Successfully updated the environment
 │ │ 
 │ │ Finalized run dependencies: this output has no run dependencies
 │ │
 │ ╰─────────────────── (took 8 seconds)
 │
 │ ╭─ Running build script
 │ │ + git -C /tmp/output/bld/rattler-build_libminc_1710251385/work status
 │ │ On branch master
 │ │ Your branch is up to date with 'origin/master'.
 │ │ Untracked files:
 │ │   (use "git add <file>..." to include in what will be committed)
 │ │ 	build_env.sh
 │ │ 	conda_build.sh
 │ │ nothing added to commit but untracked files present (use "git add" to track)
 │ │ + ls -la /tmp/output/bld/rattler-build_libminc_1710251385/work
 │ │ total 212
 │ │ drwxrwxr-x 14 me me  4096 Mar 12 14:49 .
 │ │ drwxrwxr-x  5 me me  4096 Mar 12 14:49 ..
 │ │ drwxrwxr-x  8 me me  4096 Mar 12 14:49 .git
 │ │ drwxrwxr-x  3 me me  4096 Mar 12 14:49 .github
 │ │ -rw-rw-r--  1 me me   276 Mar 12 14:49 .gitignore
 │ │ -rw-rw-r--  1 me me     0 Mar 12 14:49 .gitmodules
 │ │ -rw-rw-r--  1 me me   485 Mar 12 14:49 .travis.yml
 │ │ -rw-rw-r--  1 me me   549 Mar 12 14:49 AUTHORS
 │ │ -rw-rw-r--  1 me me 19385 Mar 12 14:49 CMakeLists.txt
 │ │ -rw-rw-r--  1 me me   490 Mar 12 14:49 COPYING
 │ │ -rw-rw-r--  1 me me   532 Mar 12 14:49 CTestConfig.cmake
 │ │ -rw-rw-r--  1 me me 43088 Mar 12 14:49 ChangeLog
 │ │ -rw-rw-r--  1 me me  1124 Mar 12 14:49 INSTALL
 │ │ -rw-rw-r--  1 me me  1545 Mar 12 14:49 LIBMINCConfig.cmake.in
 │ │ -rw-rw-r--  1 me me 31752 Mar 12 14:49 NEWS
 │ │ -rw-rw-r--  1 me me  3602 Mar 12 14:49 README
 │ │ -rw-rw-r--  1 me me  1480 Mar 12 14:49 README.release
 │ │ -rw-rw-r--  1 me me   343 Mar 12 14:49 UseLIBMINC.cmake.in
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:49 brew
 │ │ -rw-rw-r--  1 me me  4367 Mar 12 14:49 build_env.sh
 │ │ -rw-rw-r--  1 me me   330 Mar 12 14:49 check_clock_gettime.c
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:49 cmake-modules
 │ │ -rw-rw-r--  1 me me   391 Mar 12 14:49 conda_build.sh
 │ │ -rw-rw-r--  1 me me  1982 Mar 12 14:49 config.h.cmake
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:49 doc
 │ │ drwxrwxr-x  4 me me  4096 Mar 12 14:49 ezminc
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:49 libcommon
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:49 libsrc
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:49 libsrc2
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:49 nifti
 │ │ drwxrwxr-x  3 me me  4096 Mar 12 14:49 testdir
 │ │ drwxrwxr-x  8 me me  4096 Mar 12 14:49 volume_io
 │ │
 │ ╰─────────────────── (took 0 seconds)
 │
 │ ╭─ Packaging new files
 │ │ Copying done!
 │ │ Post-processing done!
 │ │ Writing metadata for package
 │ │ 
 │ │ Files in package:
 │ │   - info/about.json
 │ │   - info/hash_input.json
 │ │   - info/index.json
 │ │   - info/paths.json
 │ │   - info/recipe/recipe.yaml
 │ │   - info/recipe/rendered_recipe.yaml
 │ │   - info/recipe/variant_config.yaml
 │ │ Creating target folder "/tmp/output/linux-64"
 │ │ Compressing archive...
 │ │ Archive written to "/tmp/output/linux-64/libminc-2.3.0-hb0f4dca_0.conda"
 │ │
 │ ╰─────────────────── (took 0 seconds)
 │
 │ ╭─ Running package tests
 │ │ Removing previously cached package "/home/me/.cache/rattler/cache/pkgs/libminc-2.3.0-hb0f4dca_0"
 │ │ Creating test environment in "/tmp/output/bld/rattler-build_libminc_1710251385/work/test"
 │ │ Collecting tests from "/home/me/.cache/rattler/cache/pkgs/libminc-2.3.0-hb0f4dca_0"
 │ │
 │ ╰─────────────────── (took 0 seconds)
 │
 ╰─────────────────── (took 12 seconds)

 ╭─ Build summary
 │
 │ ╭─ Build summary for recipe: libminc-2.3.0-hb0f4dca_0
 │ │ Artifact: /tmp/output/linux-64/libminc-2.3.0-hb0f4dca_0.conda (5.05 KiB)
 │ │ Variant configuration (hash: hb0f4dca_0):
 │ │ ╭─────────────────┬──────────╮
 │ │ │ target_platform ┆ linux-64 │
 │ │ ╰─────────────────┴──────────╯
 │ │ 
 │ │ Build dependencies:
 │ │ ╭─────────────────┬──────┬──────────────┬──────────────────┬─────────────┬────────────╮
 │ │ │ Package         ┆ Spec ┆ Version      ┆ Build            ┆ Channel     ┆       Size │
 │ │ ╞═════════════════╪══════╪══════════════╪══════════════════╪═════════════╪════════════╡
 │ │ │ git             ┆ git  ┆ 2.44.0       ┆ pl5321h709897a_0 ┆ conda-forge ┆  10.21 MiB │
 │ │ │ _libgcc_mutex   ┆      ┆ 0.1          ┆ conda_forge      ┆ conda-forge ┆   2.50 KiB │
 │ │ │ _openmp_mutex   ┆      ┆ 4.5          ┆ 2_gnu            ┆ conda-forge ┆  23.07 KiB │
 │ │ │ bzip2           ┆      ┆ 1.0.8        ┆ hd590300_5       ┆ conda-forge ┆ 248.27 KiB │
 │ │ │ c-ares          ┆      ┆ 1.27.0       ┆ hd590300_0       ┆ conda-forge ┆ 159.74 KiB │
 │ │ │ ca-certificates ┆      ┆ 2024.2.2     ┆ hbcca054_0       ┆ conda-forge ┆ 151.79 KiB │
 │ │ │ curl            ┆      ┆ 8.5.0        ┆ hca28451_0       ┆ conda-forge ┆  92.67 KiB │
 │ │ │ gettext         ┆      ┆ 0.21.1       ┆ h27087fc_0       ┆ conda-forge ┆   4.12 MiB │
 │ │ │ keyutils        ┆      ┆ 1.6.1        ┆ h166bdaf_0       ┆ conda-forge ┆ 115.07 KiB │
 │ │ │ krb5            ┆      ┆ 1.21.2       ┆ h659d440_0       ┆ conda-forge ┆   1.31 MiB │
 │ │ │ libcurl         ┆      ┆ 8.5.0        ┆ hca28451_0       ┆ conda-forge ┆ 380.04 KiB │
 │ │ │ libedit         ┆      ┆ 3.1.20191231 ┆ he28a2e2_2       ┆ conda-forge ┆ 120.97 KiB │
 │ │ │ libev           ┆      ┆ 4.33         ┆ hd590300_2       ┆ conda-forge ┆ 110.12 KiB │
 │ │ │ libexpat        ┆      ┆ 2.6.1        ┆ h59595ed_0       ┆ conda-forge ┆  71.85 KiB │
 │ │ │ libgcc-ng       ┆      ┆ 13.2.0       ┆ h807b86a_5       ┆ conda-forge ┆ 752.45 KiB │
 │ │ │ libgomp         ┆      ┆ 13.2.0       ┆ h807b86a_5       ┆ conda-forge ┆ 409.91 KiB │
 │ │ │ libiconv        ┆      ┆ 1.17         ┆ hd590300_2       ┆ conda-forge ┆ 689.23 KiB │
 │ │ │ libnghttp2      ┆      ┆ 1.58.0       ┆ h47da74e_1       ┆ conda-forge ┆ 617.12 KiB │
 │ │ │ libssh2         ┆      ┆ 1.11.0       ┆ h0841786_0       ┆ conda-forge ┆ 264.78 KiB │
 │ │ │ libstdcxx-ng    ┆      ┆ 13.2.0       ┆ h7e041cc_5       ┆ conda-forge ┆   3.66 MiB │
 │ │ │ libxcrypt       ┆      ┆ 4.4.36       ┆ hd590300_1       ┆ conda-forge ┆  98.04 KiB │
 │ │ │ libzlib         ┆      ┆ 1.2.13       ┆ hd590300_5       ┆ conda-forge ┆  60.14 KiB │
 │ │ │ ncurses         ┆      ┆ 6.4          ┆ h59595ed_2       ┆ conda-forge ┆ 863.71 KiB │
 │ │ │ openssl         ┆      ┆ 3.2.1        ┆ hd590300_0       ┆ conda-forge ┆   2.73 MiB │
 │ │ │ pcre2           ┆      ┆ 10.43        ┆ hcad00b1_0       ┆ conda-forge ┆ 928.56 KiB │
 │ │ │ perl            ┆      ┆ 5.32.1       ┆ 7_hd590300_perl5 ┆ conda-forge ┆  12.73 MiB │
 │ │ │ zstd            ┆      ┆ 1.5.5        ┆ hfc55251_0       ┆ conda-forge ┆ 532.42 KiB │
 │ │ ╰─────────────────┴──────┴──────────────┴──────────────────┴─────────────┴────────────╯
 │ │ 
 │ │
 │ ╰─────────────────── (took 0 seconds)
 │
 ╰─────────────────── (took 0 seconds)
```
```
$ rattler-build build -r /tmp/recipe/ --output-dir /tmp/output
 Target platform: linux-64

 ╭─ Finding outputs from recipe
 │ Found 1 variants
 │ Build variant: libminc-2.3.0-hb0f4dca_0
 │ ╭─────────────────┬──────────╮
 │ │ Variant         ┆ Version  │
 │ ╞═════════════════╪══════════╡
 │ │ target_platform ┆ linux-64 │
 │ ╰─────────────────┴──────────╯
 │
 ╰─────────────────── (took 0 seconds)

 ╭─ Running build for recipe: libminc-2.3.0-hb0f4dca_0
 │
 │ ╭─ Fetching source code
 │ │ Fetching source from git repo: https://github.com/BIC-MNI/libminc
 │ │ Fetching repository from https://github.com/BIC-MNI/libminc at refs/tags/libminc-2-3-00
From https://github.com/BIC-MNI/libminc
 * tag                 libminc-2-3-00 -> FETCH_HEAD
 │ │ Checked out revision: 'refs/tags/libminc-2-3-00' at '4ec4e79f7e7cf41ec61e24865096316efbf90bb3'
 │ │
 │ ╰─────────────────── (took 0 seconds)
 │
 │ ╭─ Resolving environments
 │ │ 
 │ │ Resolving environment for:
 │ │   Platform: linux-64
 │ │   Channels: 
 │ │    - /tmp/output
 │ │    - conda-forge
 │ │   Specs:
 │ │    - git
 │ │     /tmp/output/noarch   [00:00:00] Using cache
 │ │ 
 │ │ ╭─────────────────┬──────────────┬──────────────────┬─────────────┬────────────╮
 │ │ │ Package         ┆ Version      ┆ Build            ┆ Channel     ┆ Size       │
 │ │ ╞═════════════════╪══════════════╪══════════════════╪═════════════╪════════════╡
 │ │ │ git             ┆ 2.44.0       ┆ pl5321h709897a_0 ┆ conda-forge ┆ 10.21 MiB  │
 │ │ │ pcre2           ┆ 10.43        ┆ hcad00b1_0       ┆ conda-forge ┆ 928.56 KiB │
 │ │ │ openssl         ┆ 3.2.1        ┆ hd590300_0       ┆ conda-forge ┆ 2.73 MiB   │
 │ │ │ libexpat        ┆ 2.6.1        ┆ h59595ed_0       ┆ conda-forge ┆ 71.85 KiB  │
 │ │ │ libzlib         ┆ 1.2.13       ┆ hd590300_5       ┆ conda-forge ┆ 60.14 KiB  │
 │ │ │ libiconv        ┆ 1.17         ┆ hd590300_2       ┆ conda-forge ┆ 689.23 KiB │
 │ │ │ bzip2           ┆ 1.0.8        ┆ hd590300_5       ┆ conda-forge ┆ 248.27 KiB │
 │ │ │ gettext         ┆ 0.21.1       ┆ h27087fc_0       ┆ conda-forge ┆ 4.12 MiB   │
 │ │ │ libgcc-ng       ┆ 13.2.0       ┆ h807b86a_5       ┆ conda-forge ┆ 752.45 KiB │
 │ │ │ _libgcc_mutex   ┆ 0.1          ┆ conda_forge      ┆ conda-forge ┆ 2.50 KiB   │
 │ │ │ _openmp_mutex   ┆ 4.5          ┆ 2_gnu            ┆ conda-forge ┆ 23.07 KiB  │
 │ │ │ libgomp         ┆ 13.2.0       ┆ h807b86a_5       ┆ conda-forge ┆ 409.91 KiB │
 │ │ │ perl            ┆ 5.32.1       ┆ 7_hd590300_perl5 ┆ conda-forge ┆ 12.73 MiB  │
 │ │ │ libxcrypt       ┆ 4.4.36       ┆ hd590300_1       ┆ conda-forge ┆ 98.04 KiB  │
 │ │ │ curl            ┆ 8.5.0        ┆ hca28451_0       ┆ conda-forge ┆ 92.67 KiB  │
 │ │ │ zstd            ┆ 1.5.5        ┆ hfc55251_0       ┆ conda-forge ┆ 532.42 KiB │
 │ │ │ libssh2         ┆ 1.11.0       ┆ h0841786_0       ┆ conda-forge ┆ 264.78 KiB │
 │ │ │ libcurl         ┆ 8.5.0        ┆ hca28451_0       ┆ conda-forge ┆ 380.04 KiB │
 │ │ │ krb5            ┆ 1.21.2       ┆ h659d440_0       ┆ conda-forge ┆ 1.31 MiB   │
 │ │ │ keyutils        ┆ 1.6.1        ┆ h166bdaf_0       ┆ conda-forge ┆ 115.07 KiB │
 │ │ │ libnghttp2      ┆ 1.58.0       ┆ h47da74e_1       ┆ conda-forge ┆ 617.12 KiB │
 │ │ │ libev           ┆ 4.33         ┆ hd590300_2       ┆ conda-forge ┆ 110.12 KiB │
 │ │ │ libedit         ┆ 3.1.20191231 ┆ he28a2e2_2       ┆ conda-forge ┆ 120.97 KiB │
 │ │ │ c-ares          ┆ 1.27.0       ┆ hd590300_0       ┆ conda-forge ┆ 159.74 KiB │
 │ │ │ ncurses         ┆ 6.4          ┆ h59595ed_2       ┆ conda-forge ┆ 863.71 KiB │
 │ │ │ libstdcxx-ng    ┆ 13.2.0       ┆ h7e041cc_5       ┆ conda-forge ┆ 3.66 MiB   │
 │ │ │ ca-certificates ┆ 2024.2.2     ┆ hbcca054_0       ┆ conda-forge ┆ 151.79 KiB │
 │ │ ╰─────────────────┴──────────────┴──────────────────┴─────────────┴────────────╯
 │ │  Successfully updated the environment
 │ │ 
 │ │ Finalized run dependencies: this output has no run dependencies
 │ │
 │ ╰─────────────────── (took 1 second)
 │
 │ ╭─ Running build script
 │ │ + git -C /tmp/output/bld/rattler-build_libminc_1710251404/work status
 │ │ On branch master
 │ │ Your branch is behind 'origin/master' by 229 commits, and can be fast-forwarded.
 │ │   (use "git pull" to update your local branch)
 │ │ Untracked files:
 │ │   (use "git add <file>..." to include in what will be committed)
 │ │ 	build_env.sh
 │ │ 	conda_build.sh
 │ │ nothing added to commit but untracked files present (use "git add" to track)
 │ │ + ls -la /tmp/output/bld/rattler-build_libminc_1710251404/work
 │ │ total 188
 │ │ drwxrwxr-x 11 me me  4096 Mar 12 14:50 .
 │ │ drwxrwxr-x  5 me me  4096 Mar 12 14:50 ..
 │ │ drwxrwxr-x  8 me me  4096 Mar 12 14:50 .git
 │ │ -rw-rw-r--  1 me me   244 Mar 12 14:50 .gitignore
 │ │ -rw-rw-r--  1 me me     0 Mar 12 14:50 .gitmodules
 │ │ -rw-rw-r--  1 me me   544 Mar 12 14:50 AUTHORS
 │ │ -rw-rw-r--  1 me me 15711 Mar 12 14:50 CMakeLists.txt
 │ │ -rw-rw-r--  1 me me   490 Mar 12 14:50 COPYING
 │ │ -rw-rw-r--  1 me me   531 Mar 12 14:50 CTestConfig.cmake
 │ │ -rw-rw-r--  1 me me 41475 Mar 12 14:50 ChangeLog
 │ │ -rw-rw-r--  1 me me  1124 Mar 12 14:50 INSTALL
 │ │ -rw-rw-r--  1 me me  1237 Mar 12 14:50 LIBMINCConfig.cmake.in
 │ │ -rw-rw-r--  1 me me 31752 Mar 12 14:50 NEWS
 │ │ -rw-rw-r--  1 me me  3153 Mar 12 14:50 README
 │ │ -rw-rw-r--  1 me me  1479 Mar 12 14:50 README.release
 │ │ -rw-rw-r--  1 me me   343 Mar 12 14:50 UseLIBMINC.cmake.in
 │ │ -rw-rw-r--  1 me me  4367 Mar 12 14:50 build_env.sh
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:50 cmake-modules
 │ │ -rw-rw-r--  1 me me   391 Mar 12 14:50 conda_build.sh
 │ │ -rw-rw-r--  1 me me  1800 Mar 12 14:50 config.h.cmake
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:50 doc
 │ │ drwxrwxr-x  4 me me  4096 Mar 12 14:50 ezminc
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:50 libsrc
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:50 libsrc2
 │ │ drwxrwxr-x  2 me me  4096 Mar 12 14:50 nifti
 │ │ drwxrwxr-x  3 me me  4096 Mar 12 14:50 testdir
 │ │ drwxrwxr-x  8 me me  4096 Mar 12 14:50 volume_io
 │ │
 │ ╰─────────────────── (took 0 seconds)
 │
 │ ╭─ Packaging new files
 │ │ Copying done!
 │ │ Post-processing done!
 │ │ Writing metadata for package
 │ │ 
 │ │ Files in package:
 │ │   - info/about.json
 │ │   - info/hash_input.json
 │ │   - info/index.json
 │ │   - info/paths.json
 │ │   - info/recipe/recipe.yaml
 │ │   - info/recipe/rendered_recipe.yaml
 │ │   - info/recipe/variant_config.yaml
 │ │ Creating target folder "/tmp/output/linux-64"
 │ │ Compressing archive...
 │ │ Archive written to "/tmp/output/linux-64/libminc-2.3.0-hb0f4dca_0.conda"
 │ │
 │ ╰─────────────────── (took 0 seconds)
 │
 │ ╭─ Running package tests
 │ │ Removing previously cached package "/home/me/.cache/rattler/cache/pkgs/libminc-2.3.0-hb0f4dca_0"
 │ │ Creating test environment in "/tmp/output/bld/rattler-build_libminc_1710251404/work/test"
 │ │ Collecting tests from "/home/me/.cache/rattler/cache/pkgs/libminc-2.3.0-hb0f4dca_0"
 │ │
 │ ╰─────────────────── (took 0 seconds)
 │
 ╰─────────────────── (took 1 second)

 ╭─ Build summary
 │
 │ ╭─ Build summary for recipe: libminc-2.3.0-hb0f4dca_0
 │ │ Artifact: /tmp/output/linux-64/libminc-2.3.0-hb0f4dca_0.conda (5.03 KiB)
 │ │ Variant configuration (hash: hb0f4dca_0):
 │ │ ╭─────────────────┬──────────╮
 │ │ │ target_platform ┆ linux-64 │
 │ │ ╰─────────────────┴──────────╯
 │ │ 
 │ │ Build dependencies:
 │ │ ╭─────────────────┬──────┬──────────────┬──────────────────┬─────────────┬────────────╮
 │ │ │ Package         ┆ Spec ┆ Version      ┆ Build            ┆ Channel     ┆       Size │
 │ │ ╞═════════════════╪══════╪══════════════╪══════════════════╪═════════════╪════════════╡
 │ │ │ git             ┆ git  ┆ 2.44.0       ┆ pl5321h709897a_0 ┆ conda-forge ┆  10.21 MiB │
 │ │ │ _libgcc_mutex   ┆      ┆ 0.1          ┆ conda_forge      ┆ conda-forge ┆   2.50 KiB │
 │ │ │ _openmp_mutex   ┆      ┆ 4.5          ┆ 2_gnu            ┆ conda-forge ┆  23.07 KiB │
 │ │ │ bzip2           ┆      ┆ 1.0.8        ┆ hd590300_5       ┆ conda-forge ┆ 248.27 KiB │
 │ │ │ c-ares          ┆      ┆ 1.27.0       ┆ hd590300_0       ┆ conda-forge ┆ 159.74 KiB │
 │ │ │ ca-certificates ┆      ┆ 2024.2.2     ┆ hbcca054_0       ┆ conda-forge ┆ 151.79 KiB │
 │ │ │ curl            ┆      ┆ 8.5.0        ┆ hca28451_0       ┆ conda-forge ┆  92.67 KiB │
 │ │ │ gettext         ┆      ┆ 0.21.1       ┆ h27087fc_0       ┆ conda-forge ┆   4.12 MiB │
 │ │ │ keyutils        ┆      ┆ 1.6.1        ┆ h166bdaf_0       ┆ conda-forge ┆ 115.07 KiB │
 │ │ │ krb5            ┆      ┆ 1.21.2       ┆ h659d440_0       ┆ conda-forge ┆   1.31 MiB │
 │ │ │ libcurl         ┆      ┆ 8.5.0        ┆ hca28451_0       ┆ conda-forge ┆ 380.04 KiB │
 │ │ │ libedit         ┆      ┆ 3.1.20191231 ┆ he28a2e2_2       ┆ conda-forge ┆ 120.97 KiB │
 │ │ │ libev           ┆      ┆ 4.33         ┆ hd590300_2       ┆ conda-forge ┆ 110.12 KiB │
 │ │ │ libexpat        ┆      ┆ 2.6.1        ┆ h59595ed_0       ┆ conda-forge ┆  71.85 KiB │
 │ │ │ libgcc-ng       ┆      ┆ 13.2.0       ┆ h807b86a_5       ┆ conda-forge ┆ 752.45 KiB │
 │ │ │ libgomp         ┆      ┆ 13.2.0       ┆ h807b86a_5       ┆ conda-forge ┆ 409.91 KiB │
 │ │ │ libiconv        ┆      ┆ 1.17         ┆ hd590300_2       ┆ conda-forge ┆ 689.23 KiB │
 │ │ │ libnghttp2      ┆      ┆ 1.58.0       ┆ h47da74e_1       ┆ conda-forge ┆ 617.12 KiB │
 │ │ │ libssh2         ┆      ┆ 1.11.0       ┆ h0841786_0       ┆ conda-forge ┆ 264.78 KiB │
 │ │ │ libstdcxx-ng    ┆      ┆ 13.2.0       ┆ h7e041cc_5       ┆ conda-forge ┆   3.66 MiB │
 │ │ │ libxcrypt       ┆      ┆ 4.4.36       ┆ hd590300_1       ┆ conda-forge ┆  98.04 KiB │
 │ │ │ libzlib         ┆      ┆ 1.2.13       ┆ hd590300_5       ┆ conda-forge ┆  60.14 KiB │
 │ │ │ ncurses         ┆      ┆ 6.4          ┆ h59595ed_2       ┆ conda-forge ┆ 863.71 KiB │
 │ │ │ openssl         ┆      ┆ 3.2.1        ┆ hd590300_0       ┆ conda-forge ┆   2.73 MiB │
 │ │ │ pcre2           ┆      ┆ 10.43        ┆ hcad00b1_0       ┆ conda-forge ┆ 928.56 KiB │
 │ │ │ perl            ┆      ┆ 5.32.1       ┆ 7_hd590300_perl5 ┆ conda-forge ┆  12.73 MiB │
 │ │ │ zstd            ┆      ┆ 1.5.5        ┆ hfc55251_0       ┆ conda-forge ┆ 532.42 KiB │
 │ │ ╰─────────────────┴──────┴──────────────┴──────────────────┴─────────────┴────────────╯
 │ │ 
 │ │
 │ ╰─────────────────── (took 0 seconds)
 │
 ╰─────────────────── (took 0 seconds)
```

**Comment by wolfv** [comment]
I've opened #755 to fix this.

**Comment by wolfv** [comment]
This is fixed in 0.14.1, thanks!
